binstar_client/__init__.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, token=None, domain='https://api.anaconda.org', verify=True, *
 
         if domain.endswith('/'):
             domain = domain[:-1]
+        if not domain.startswith(('http://', 'https://')):
```

Comment:
This is not working. Apparently `startswith` performs an **or** when its called with a tuple, so it won't change to `https` when a domain with `http` is given.

Here's an example:

```python
>>> domain = 'http://some-url.com'
>>> if not domain.startswith(('http://', 'https://')):
...     domain = 'https://' + domain
...
>>> domain
'http://some-url.com'
```